pmm-app/src/pmm-qan/panel/components/Filters/Filters.styles.ts

```diff
@@ -3,20 +3,19 @@ import { css } from 'emotion';
 export const styles = {
   getFiltersWrapper: (height) => css`
     border: 1px solid rgb(40, 40, 40);
-    padding: 5px 16px !important;
-    overflow-y: scroll;
     height: ${height}px;
   `,
+  filterContent: css`
+    padding: 10px 16px !important;
```

Comment:
Do we really need this `!important`?

Reply:
@atymchuk added but I testes myself and yes seems like we need.

Reply:
Yes, because of antd's styles again!